deals/tasks.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+from celery import shared_task
+
+from auto_show.models import AutoShow
+from car.models import CarInstance
+from deals.models import Deal
+from producer.models import Producer
+
+
+@shared_task
+def sale_car_from_producer():
+    # select one auto_show
+    for auto_show in AutoShow.objects.all():
+
+        # select auto_show's balance
+        auto_show_balance = auto_show.balance
+
+        # select auto_show's wish_car and transform it from TextField to list
+        list_wish_car = auto_show.wish_car.split('  ')
+
+        # select fields for comparing
+        for elem in list_wish_car:
+            b = json.loads(elem)
+            brand_wish_car = b.get("brand")
+            model_wish_car = b.get("model")
+            price_wish_car = b.get("price")
+
+            # check price
+            if price_wish_car != "None" and price_wish_car is not None:
```

Comment:
if price field is required don't use b.get("price")
Use b["price"], because it is better to send an exception, that something wasn't passed for debugging purposes